services/ux-backend/handlers/onboardingtokens/handler.go

```diff
@@ -16,14 +18,35 @@ const (
 func HandleMessage(w http.ResponseWriter, r *http.Request, tokenLifetimeInHours int) {
 	switch r.Method {
 	case "POST":
-		handlePost(w, tokenLifetimeInHours)
+		type Message struct {
+			Quota string `json:"quota"`
+		}
+		var quota Message
```

Comment:
1. I don't think we should be processing the body before sending it to `handlePost` cause in the switch you are only deciding the method
2. No need to create a type if we are not going to initialize it, can get away w/ var. Unrelated one, for encoding & decoding you only need fields to be exported not necessarily the struct.
```suggestion
		var quota = struct { Quota string `json:"quota"`}{}
```